homeassistant/components/deconz/light.py

```diff
@@ -105,6 +105,23 @@ class SetStateAttributes(TypedDict, total=False):
     xy: tuple[float, float]
 
 
+def update_color_state(
+    group: Group, lights: list[Light], override: bool = False
+) -> None:
+    """Sync group color state with light."""
+    data = {
```

Comment:
Doesn't this mean that last brightness, last CT etc wins ?

Reply:
Yes that is true, but that hasn't changed from before (except it was part of the library) behaviour hasn't changed, the real change here is `group.raw["action"] = cast(TypedGroupAction, data)` where all data from all lights have been considered and then overriding any existing data in the group.